src/components/LocationSearch/LocationSearch.jsx

```diff
@@ -1,7 +1,7 @@
 import { useState } from "react";
 import PropTypes from "prop-types";
-import Close from "../../assets/icons/close.png";
-import Search from "../../assets/icons/search.png";
+import { X } from "lucide-react";
+import { Search } from "lucide-react";
```

Comment:
You're importing from the same library, so you can condense:

```suggestion
import { X, Search } from "lucide-react";
```